packages/blockchain-bindings/src/contracts.ts

```diff
@@ -3,8 +3,8 @@ import { EthereumAddress } from '@lens-protocol/shared-kernel';
 import { Contract } from 'ethers';
 
 import erc20Abi from './abi/erc-20.json';
-import { ERC20 } from './types/ERC20';
+import { IErc20Contract } from './types';
 
 export function erc20(address: EthereumAddress, provider: Provider) {
-  return new Contract(address, erc20Abi, provider) as unknown as ERC20;
+  return new Contract(address, erc20Abi, provider) as unknown as IErc20Contract & Contract;
```

Comment:
I had to change the type from the generated to a custom one because there are two problems with the generated types:
- The `overrides` type of the write functions doesn't have all the options that we use, like `maxFeePerGas` or `maxPriorityFeePerGas`
- The type doesn't have `estimateGas` and `populateTransaction` methods that `ethers` exports in their contracts, and they don't export a generic or separated write/read methods types for me to be able to extend the generated type.

Reply:
I can see that the generator has something called `ContractContext` with `estimateGas` and `populateTransaction` (they are not fully typed as they are using indexed types).
But still, we lack the `maxFeePerGas` and `maxPriorityFeePerGas` from EIP1559.
@joshstevens19 is that something that we can easily add to your codegen to not added custom code into the sdk?